scripts/job_bq_webdataset.sh

```
@@ -0,0 +1,144 @@
+echo ""
+
+# ── Setup Lustre output with HDD stripe ──────────────────────────────────────
+for SPLIT in train val test; do
+    mkdir -p "${OUTPUT_DIR}/${SPLIT}"
+    lfs setstripe -c -1 -S 4m -p ddn_hdd "${OUTPUT_DIR}/${SPLIT}" 2>/dev/null || true
+done
+echo "Output dir:  ${OUTPUT_DIR}"
+echo ""
+
+# ── Python environment ────────────────────────────────────────────────────────
+cd /project/6068129/melabbas/ami-ml
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)